jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -127,6 +128,20 @@ public PermissionConfiguration() {}
     }
   }
 
+  /** Configuration for {@code platform} parameter. Defaults to amd64/linux. * */
+  public static class PlatformsConfiguration {
+    @Parameter private String os = "linux";
+    @Parameter private String architecture = "amd64";
```

Comment:
As discussed in the design doc, if we are going to raise an error when one of these are missing, I think we have to make these `@Nullable` and not assign default values.

Need to add `@Nullable` in getters too.

Reply:
Done